tsdb/compact_test.go

```diff
@@ -933,6 +934,31 @@ func TestCompaction_populateBlock(t *testing.T) {
 				},
 			},
 		},
+		{
+			title:          "Populate from mixed type series and expect sample inside the interval only", // regression test for populateWithDelChunkSeriesIterator failing to set minTime on chunks
```

Comment:
Spelling nit: Could you end the title string with a period, to be consistent with the other titles? And the comment would also read more easily if it started with a capital letter end ended with a period. Maybe even put that comment on its own line because the line is now very long in total…